### PR TITLE
Dummy shopping cart orders

### PR DESCRIPTION
**Pull request by zeljadin**
**Add:**

confirmation.hbs
order-details.hbs
summary.hbs

**Tweak styling:**
caret.scss
buttons.scss
cards.scss
dropdown.scss
navbar.scss
notifications.scss
progress-bar.scss
tables.scss
text.scss
variables.scss

**Add responsive across all dummy pages.**

### CODE REVIEW

File: app/styles/ebf-design-system/themes/telekom/utilities/_borders.scss [hunk 1]
```diff
@@ -4,6 +4,46 @@
...
+}
+
+.border-bottom-0 {
+  border-bottom: 0 !important;
+}
+
+.border-left-0 {
+  border-left: 0 !important;
+}
+
```
Comment (vladaspasic) [inline app/styles/ebf-design-system/themes/telekom/utilities/_borders.scss:46]: Isn't this already in Bootstrap?

File: app/styles/ebf-design-system/themes/telekom/components/_type.scss [hunk 2]
```diff
@@ -110,6 +110,27 @@ mark,
...
+}
+
+.text-justify {
+  text-align: justify;
+}
+
+.text-nowrap {
+  white-space: nowrap;
+}
+
```
Comment (vladaspasic) [inline app/styles/ebf-design-system/themes/telekom/components/_type.scss:133]: Isn't this already in Bootstrap?

File: app/styles/ebf-design-system/themes/telekom/utilities/_borders.scss (hunk 1 above)
Comment (zeljadin) [reply app/styles/ebf-design-system/themes/telekom/utilities/_borders.scss:46]: Native Bootstrap styles were excluded for these rules

File: app/styles/ebf-design-system/themes/telekom/components/_type.scss (hunk 2 above)
Comment (zeljadin) [reply app/styles/ebf-design-system/themes/telekom/components/_type.scss:133]: Native Bootstrap styles were excluded for these rules

**Pull request merged**